framework/doc/content/framework/contributing.md

```diff
@@ -13,6 +13,10 @@ engaging in the community will certainly be beneficial.
 
 !alert-end!
 
+!alert note title=Authorship in MOOSE-development paper
```

Comment:
```suggestion
!alert note title=Authorship in MOOSE-development papers
```

Reply:
To help denote that there are multiple kinds of MOOSE papers, including a framework paper, module papers, etc.